integration/matchers/match_toml_content.go

```diff
@@ -0,0 +1,46 @@
+package matchers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/onsi/gomega/types"
+	. "github.com/paketo-buildpacks/packit/v2/matchers"
+)
+
+func MatchTomlContent(expectedFilePath string) types.GomegaMatcher {
+	return &matchTomlContentMatcher{
+		expectedFilePath: expectedFilePath,
+	}
+}
```

Comment:
We added this matcher so that we don't have to manually open both `buildpack.toml` files over and over again. It essentially just opens the files, and then passed the content into the existent occam MatchTOML matcher.